supervisely/api/entity_annotation/figure_api.py

```diff
@@ -373,6 +373,45 @@ def _append_bulk(
                 figure_id = resp_obj[ApiField.ID]
                 key_id_map.add_figure(key, figure_id)
 
+    def create_bulk(
+        self,
+        figures_json: List[dict],
+        entity_id: int = None,
+        dataset_id: int = None,
+        batch_size=200,
+    ) -> List[int]:
```

Comment:
Maybe also add to api.annotation as add_labels_bulk or smth

Reply:
Will be added in another request